source/usage-examples/count.txt

```diff
@@ -3,3 +3,47 @@ Count Documents
 ===============
 
 .. default-domain:: mongodb
+
+You can count the number of documents in a collection using the
+``EstimatedDocumentCount()`` and ``CountDocuments()`` methods. 
+
+The following example uses the ``EstimatedDocumentCount()`` method to
+count the number of documents in the ``movies`` collection. Then, the
+example uses the ``CountDocuments()`` method to count the number of
+documents in the ``movies`` collection where the ``countries`` field
+contains "China".
+
+.. include:: /includes/usage-examples/run-example-tip.rst
+
+.. literalinclude:: /includes/usage-examples/code-snippets/count.go
+   :start-after: begin countDocuments
+   :end-before: end countDocuments
+   :emphasize-lines: 4-5
+   :language: go
+   :dedent:
+
+Click `here <{+example+}/count.go>`__ to see a fully runnable example.
+
+Expected Result
+---------------
+
+After running the preceding code snippet, you should see the following:
+
+- There are about ``23541`` documents in the ``movies`` collection  
+- There are ``303`` documents in the ``movies`` collection from "China"
```

Comment:
I think these print statements differ from the ones on L50 and L51 of `count.go`.

Reply:
Since I don't show the print statements from the go file, I generalized it to what each variable should contain in terms of value.